Dockerfile

```diff
@@ -1,6 +1,6 @@
 FROM docker.io/centos:centos7
```

Comment:
A small comment - can we move to centos8

Reply:
let's do it in a different patch though

Reply:
Sure, I will submit a PR for this.